CLA/signatures/v1.0/cla.json

```diff
@@ -1,92 +0,0 @@
-{
-  "signedContributors": [
-    {
-      "name": "mfcorgi",
-      "id": 46997613,
-      "comment_id": 812102542,
-      "created_at": "2021-04-01T18:49:19Z",
-      "repoId": 299162738,
-      "pullRequestNo": 18
-    },
-    {
-      "name": "mairadanielaferrari",
-      "id": 3629162,
-      "comment_id": 812106753,
-      "created_at": "2021-04-01T18:57:06Z",
-      "repoId": 299162738,
-      "pullRequestNo": 18
-    },
-    {
-      "name": "mrbiggred",
-      "id": 17273852,
-      "comment_id": 866387936,
-      "created_at": "2021-06-22T22:51:04Z",
-      "repoId": 299162738,
-      "pullRequestNo": 30
-    },
-    {
-      "name": "lizcorgi",
-      "id": 49960292,
-      "comment_id": 868154787,
-      "created_at": "2021-06-25T02:29:29Z",
-      "repoId": 299162738,
-      "pullRequestNo": 29
-    },
-    {
-      "name": "dfar-io",
-      "id": 564106,
-      "comment_id": 900664638,
-      "created_at": "2021-08-17T22:09:59Z",
-      "repoId": 299162738,
-      "pullRequestNo": 40
-    },
-    {
-      "name": "marnen",
-      "id": 34378,
-      "comment_id": 1057114791,
-      "created_at": "2022-03-02T16:23:05Z",
-      "repoId": 299162738,
-      "pullRequestNo": 48
-    },
-    {
-      "name": "CodeMouse92",
-      "id": 7471402,
-      "comment_id": 1118695063,
-      "created_at": "2022-05-05T15:25:24Z",
-      "repoId": 299162738,
-      "pullRequestNo": 64
-    },
-    {
-      "name": "mscottford",
-      "id": 21340,
-      "comment_id": 1129187371,
-      "created_at": "2022-05-17T18:28:40Z",
-      "repoId": 299162738,
-      "pullRequestNo": 67
-    },
-    {
-      "name": "donabp",
-      "id": 50669675,
-      "comment_id": 1145293393,
-      "created_at": "2022-06-02T20:07:33Z",
-      "repoId": 299162738,
-      "pullRequestNo": 88
-    },
-    {
-      "name": "donabp",
-      "id": 50669675,
-      "comment_id": 1145336878,
-      "created_at": "2022-06-02T21:00:22Z",
-      "repoId": 299162738,
-      "pullRequestNo": 88
-    },
-    {
-      "name": "edwinkortman",
-      "id": 7047894,
-      "comment_id": 1154846393,
-      "created_at": "2022-06-14T07:58:05Z",
-      "repoId": 299162738,
-      "pullRequestNo": 99
-    }
-  ]
-}
```

Comment:
If we're moving in the direction of having a single repository with CLA signatures for all of the projects, then I think it makes sense to merge the signature files together. That way we have a clear match between commits and signatures. I wouldn't want to create a potential issue where there are commits that don't appear to be covered by signatures.

Reply:
Ahh, that's a good point. I'll merge them.